lib/components/form/date-time-modal.js

```diff
@@ -46,16 +46,17 @@ class DateTimeModal extends Component {
 }
 
 const mapStateToProps = (state, ownProps) => {
-  const { departArrive, date, time } = state.otp.currentQuery
+  const { date, departArrive, time } = state.otp.currentQuery
   const config = state.otp.config
   return {
     config,
-    departArrive,
     date,
+    departArrive,
     time,
     // These props below are for legacy browsers (see render method above).
-    timeFormatLegacy: coreUtils.time.getTimeFormat(config),
-    dateFormatLegacy: coreUtils.time.getDateFormat(config)
+    // eslint-disable-next-line sort-keys
```

Comment:
Yeah, this should be used in https://github.com/opentripplanner/otp-react-redux/pull/390/files#r666591987.